lib/Logger.d.ts

```diff
@@ -0,0 +1,13 @@
+declare type LogData = {
+    message: string;
+    level: 'alert' | 'info';
+};
+
+/**
+ * Register the logging callback
+ *
+ * @param {Function} callback
```

Comment:
Let's remove all JSDoc types from @param and @returns in .d.ts files.

Reply:
Forgot about this one, but I'm leaving the JSDoc type temporally for some of the properties that are not typed yet.